q2cli/tests/test_cli.py

```diff
@@ -569,7 +569,7 @@ def test_invalid_metadata_merge(self):
 
 
 class TestMetadataColumnSupport(MetadataTestsBase):
-    def test_required_missing(self):
+    def test_required_missing_file(self):
```

Comment:
yeah that's fair enough @Oddant1, can't hurt to add a test for that and maybe it will help us catch something dumb that we aren't expecting in the future. i'll get that added!